Kitodo/src/main/java/org/kitodo/services/data/UserService.java

```diff
@@ -726,18 +726,6 @@ public boolean isLoginValid(String login, String filePath) {
         return true;
     }
 
-    /**
-     * Table size.
-     *
-     * @return table size
-     */
-    public Integer getTableSize(User user) {
```

Comment:
Why did you move it to the user?

Reply:
I didn't move it. It was already there as it takes data from database. All pages are using user.getTableSize for data tables, so this one was not used at all.